wordpress-mobile/WordPress-iOS
# New Media Picker: Updated the media picker keyboard sizing logic

Fixes #7756

This PR updates the inline picker height logic that was introduced as part of #7723. A couple of changes were made:

1. The `currentKeyboardFrame`'s  coord system is converted from the window's thus providing more accurate values.
2. A computed var (`isExternalKeyboardConnected`) was added to determine if an external keyboard is currently connected.
3. The inline picker's height calculation now varies depending on if the external keyboard is present or not.

To test:

On the Sim and/or device, build and run the app.  Try a few of the following scenarios:

* (iPad with an external keyboard) Edit a post and bring up the inline picker — make sure the picker's height is appropriate. Try again, but disconnect the keyboard while the picker is visible.
* (iPad with soft keyboard) Edit a post and bring up the inline picker — make sure the picker's height is appropriate. Try again, but connect the keyboard while the picker is visible.
* Make sure iPhones still work as expected.
* Retest the fixes from #7710...make sure they all still work.

Needs review: @frosty (Thank you!!!!!1 🎉 🙇 )
